snowpack/src/build/build-import-proxy.ts

```diff
@@ -238,26 +237,9 @@ async function generateCssModuleImportProxy({
   hmr: boolean;
   config: SnowpackConfig;
 }) {
-  let moduleJson: string | undefined;
-  const processor = postCss([
-    postCssModules({
-      getJSON: (_, json) => {
-        moduleJson = json;
-      },
-    }),
-  ]);
-  const result = await processor.process(code, {
-    from: url,
-    to: url + '.proxy.js',
-  });
-  // log any warnings that happened.
-  result
-    .warnings()
-    .forEach((element) => logger.warn(`${url} - ${element.text}`, {name: 'snowpack:cssmodules'}));
-  // return the JS+CSS proxy file.
```

Comment:
First step in this PR is to centralize the CSS Module action, and make it happen 1 step earlier in the CSS. Now we persist the CSS Module results in build & SSR, rather than living only in the proxy file.